packages/gatsby-remark-images/src/index.js

```diff
@@ -328,7 +338,7 @@ module.exports = (
                 return resolve()
               }
 
-              const fileType = formattedImgTag.url.slice(-3)
+              const fileType = getImageInfo(formattedImgTag.url).ext
```

Comment:
`path.extname` returns a file extension with a dot. In other words, given `const file = 'asdfasdf.png'`

- `path.extname(file) === '.png'`
- `file.slice(-3) === 'png'`

Wouldn't this then break the subsequent checks?

Reply:
Oh, yes, sorry!

Reply:
That should now be resolved by d10a826. Thanks so much for you guidance throughout this PR!